### PR TITLE
Catwalk doc

### LINKED ISSUE #445: Add catwalk documentation on the website
Descibe
* Why we need the tool
* What it could be used for
* How it should be used.

### PR DESCRIPTION
**Pull request by sublibra**
Adding usage info in catwalk. Opted out on describing more about the tool since we discussed and decided to go for adding this in-tool instead. This work will be tracked in:
https://github.com/qlik-oss/catwalk/issues/15

Fixes #445

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit e7cd096fb5e60a45d2296f7739d367e3bacc40e7

https://deploy-preview-454--core-website.netlify.com

File: docs/tooling/catwalk.md [hunk 1]
```diff
@@ -14,5 +14,19 @@ within.
 
 You can use catwalk in the following ways:
 
-* provide a link to an engine web socket at [catwalk](https://catwalk.core.qlik.com/) to view your Qlik Core app
```
Comment (wennmo) [inline docs/tooling/catwalk.md]: Do we recommend using `http` now for catwalk because of the browser issues?

File: docs/tooling/catwalk.md [hunk 2]
```diff
@@ -14,5 +14,19 @@ within.
...
-* clone the [catwalk repo](https://github.com/qlik-oss/catwalk) and run the tool locally
\ No newline at end of file
+* provide a link to an engine web socket at [catwalk](http://catwalk.core.qlik.com/)
+* open an application from corectl by `corectl catwalk --app my-app.qvf`
+* clone the [catwalk repo](https://github.com/qlik-oss/catwalk) and run the tool locally
+
+If no web socket configuration is provided, catwalk defaults to localhost:9076.
+
+!!! Note
+    The Qlik application's content is transferred directly from the engine to the web browser through a web socket. I.e.
```
Comment (wennmo) [inline docs/tooling/catwalk.md]: web socket -> websocket

File: docs/tooling/catwalk.md [hunk 3]
```diff
@@ -14,5 +14,19 @@ within.
...
+    no data is sent to Qlik or any 3:rd party. Beside this Qlik collects anonymous usage data such as browser version
```
Comment (wennmo) [inline docs/tooling/catwalk.md]: Maybe specify that we are using Google Analytics and not some other way of collecting data usage. Makes it less suspicious imo.

File: docs/tooling/catwalk.md [hunk 4]
```diff
@@ -14,5 +14,19 @@ within.
...
+    and catwalk exceptions.
+
+## Best practices and techniques
+
+Catwalk provides an alternative view of the Qlik application's data model compared to existing products but best
```
Comment (wennmo) [inline docs/tooling/catwalk.md]: This sentence is a bit strange when reading. Maybe change to `Catwalk provides an alternative view of the Qlik application's data model compared to existing products. However best practices and data modelling techniques are still the same and can be found on the...`

File: docs/tooling/catwalk.md (hunk 1 above)
Comment (sublibra) [reply docs/tooling/catwalk.md]: Not sure at all. It was a guess from me to allow for more browsers @axelssonHakan - any thoughts?

Comment (axelssonHakan) [reply docs/tooling/catwalk.md]: Nope - default should be https but we should add logic that if someone triies to connect to ws://... we should redirect the url to http://

wss:// => https://
ws:// => http://

Comment (sublibra) [reply docs/tooling/catwalk.md]: I have added this as a new feature
https://github.com/qlik-oss/catwalk/issues/123

Will reword accordingly

**Pull request merged**